kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorIT.java

```diff
@@ -583,12 +653,24 @@ private boolean fileRowMatchesExpectedRow(JsonNode fileRow, Struct expectedRow)
    * @param S3FileKey the object file key
    * @return the extension, may be .avro, .json, or .snappy.parquet,
    */
-  private String getExtensionFromKey(String S3FileKey) {
-    String[] tokens = S3FileKey.split("\\.", 2);
-    if (tokens.length < 2) {
-      throw new RuntimeException("Could not parse extension from filename.");
+  private static String getExtensionFromKey(String S3FileKey) {
```

Comment:
Not new, but arguments should not start with uppercase.

Reply:
Even if it's a proper name such as "S3"?

Reply:
Yes, the definition of lowercase camelcase is fairly deterministic https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Reply:
done